main.rb

```diff
@@ -1,47 +1,14 @@
 require './app'
```

Comment:
- [ ] It would be better to change the `require` to `require_relative` since we are referencing files in our current directory